BuildResidentialHPXML/measure.rb

```diff
@@ -2963,169 +2963,113 @@ def validate_arguments(args)
-    errors << "heating_system_type=#{args[:heating_system_type]} and heat_pump_type=#{args[:heat_pump_type]} and heating_system_2_type=#{args[:heating_system_2_type]}" if error
+    errors << 'A second heating system was specified without a primary heating system.' if error
 
-    # apartment/sfa unit with no geometry_building_num_units
     error = ((args[:geometry_unit_type] == HPXML::ResidentialTypeApartment) || (args[:geometry_unit_type] == HPXML::ResidentialTypeSFA)) && !args[:geometry_building_num_units].is_initialized
-    errors << "geometry_unit_type=#{args[:geometry_unit_type]} and geometry_building_num_units=not provided" if error
+    errors << 'Did not specify the number of units in the building for single-family attached or apartment units.' if error
 
-    # apartment with more than 1 above grade floor
     error = (args[:geometry_unit_type] == HPXML::ResidentialTypeApartment) && (args[:geometry_unit_num_floors_above_grade] > 1)
-    errors << 'Apartment units can only have one above grade floor' if error
+    errors << 'Apartment units can only have one above-grade floor.' if error
 
-    # detached with adiabatic wall(s)
     error = (args[:geometry_unit_type] == HPXML::ResidentialTypeSFD) && (args[:geometry_unit_left_wall_is_adiabatic] || args[:geometry_unit_right_wall_is_adiabatic] || args[:geometry_unit_front_wall_is_adiabatic] || args[:geometry_unit_back_wall_is_adiabatic] || (args[:geometry_attic_type] == HPXML::AtticTypeBelowApartment) || (args[:geometry_foundation_type] == HPXML::FoundationTypeAboveApartment))
-    errors << 'No adiabatic surfaces can be applied to single-family detached homes' if error
+    errors << 'No adiabatic surfaces can be applied to single-family detached homes.' if error
 
-    # crawlspace or unconditioned basement with foundation wall and ceiling insulation
     warning = [HPXML::FoundationTypeCrawlspaceVented, HPXML::FoundationTypeCrawlspaceUnvented, HPXML::FoundationTypeBasementUnconditioned].include?(args[:geometry_foundation_type]) && ((args[:foundation_wall_insulation_r] > 0) || args[:foundation_wall_assembly_r].is_initialized) && (args[:floor_over_foundation_assembly_r] > 2.1)
-    if warning
-      warning = "geometry_foundation_type=#{args[:geometry_foundation_type]}"
-      if args[:foundation_wall_insulation_r] > 0
-        warning += " and foundation_wall_insulation_r=#{args[:foundation_wall_insulation_r]}"
-      end
-      if args[:foundation_wall_assembly_r].is_initialized
-        warning += " and foundation_wall_assembly_r=#{args[:foundation_wall_assembly_r].get}"
-      end
-      if args[:floor_over_foundation_assembly_r] > 2.1
-        warning += " and floor_over_foundation_assembly_r=#{args[:floor_over_foundation_assembly_r]}"
-      end
-      warnings << warning
-    end
+    warnings << 'Specified unconditioned basement/crawlspace foundation type with wall and ceiling insulation.' if warning
```

Comment:
```suggestion
    warnings << 'Home with unconditioned basement/crawlspace foundation type has both foundation wall insulation and floor insulation.' if warning
```

Reply:
We should align as much as possible with the terminology of floor => surface between living space and foundation space and ceiling => surface between living space and attic. That is, floor and ceiling are viewed from the living space perspective, which is consistent with the measure arguments (e.g., `floor_over_foundation_assembly_r` and `ceiling_assembly_r`).